src/System.Runtime.Serialization.Formatters/tests/TypeSerializableValue.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+
+namespace System.Runtime.Serialization.Formatters.Tests
+{
+    public readonly struct TypeSerializableValue
+    {
+        public readonly string Base64Blob;
+
+        // This is the minimum version, when the blob changed.
+        public readonly TargetFrameworkMoniker Platform;
+
+        public TypeSerializableValue(string base64Blob, TargetFrameworkMoniker platform)
+        {
+            Base64Blob = base64Blob;
+            Platform = platform;
+        }
+
+        public static int GetPlatformIndex(TypeSerializableValue[] blobs)
+        {
+            var blobList = blobs.ToList();
+            int index;
```

Comment:
That need doesn't need to be outside of the if clauses. We usually define types where we use them.